firmware/components/configuration/configuration.h

```diff
@@ -10,7 +10,7 @@
 #define STAFF_ROWS (7)
 #define STAFF_VALS (5)
 
-#define PLAYBACK_PERIOD_DEFAULT (500000)
+#define PLAYBACK_PERIOD_DEFAULT (600000) //Started at 500000
```

Comment:
Just a little nitpick - no need to comment what the value used to be, that is all tracked in the version control ;)